subprojects/docs/src/docs/release/notes.md

```diff
@@ -41,9 +41,9 @@ The following are the newly deprecated items in this Gradle release. If you have
 
 ## Potential breaking changes
 
-<!--
-### Example breaking change
--->
+### Changes to previously deprecated APIs
+
+ - The `Signature` methods `getToSignArtifact()` and `setFile(File)` are removed. 
```

Comment:
Nit: Use FQN for removed methods so that users could easily trace what exactly was removed.

Reply:
Good idea! Done in 9c525e5e27b147e3fef4671c02232ef8faa726a6.